### ISSUE TITLE
Spyder error: a general error while running any code

### ISSUE THREAD

**Issue by egesoydemir** [spyder-ide/spyder#7200]
While running any code (in my case parsing an rdf file), it runs well however, every time I run a code I get this error:

  File "/anaconda3/lib/python3.6/site-packages/qtconsole/base_frontend_mixin.py", line 163, in _dispatch
    handler(msg)
  File "/anaconda3/lib/python3.6/site-packages/spyder/widgets/ipythonconsole/namespacebrowser.py", line 220, in _handle_execute_reply
    self.handle_exec_method(msg)
  File "/anaconda3/lib/python3.6/site-packages/spyder/widgets/ipythonconsole/shell.py", line 354, in handle_exec_method
    view = ast.literal_eval(literal)
  File "/anaconda3/lib/python3.6/ast.py", line 85, in literal_eval
    return _convert(node_or_string)
  File "/anaconda3/lib/python3.6/ast.py", line 66, in _convert
    in zip(node.keys, node.values))
  File "/anaconda3/lib/python3.6/ast.py", line 65, in <genexpr>
    return dict((_convert(k), _convert(v)) for k, v
  File "/anaconda3/lib/python3.6/ast.py", line 66, in _convert
    in zip(node.keys, node.values))
  File "/anaconda3/lib/python3.6/ast.py", line 65, in <genexpr>
    return dict((_convert(k), _convert(v)) for k, v
  File "/anaconda3/lib/python3.6/ast.py", line 84, in _convert
    raise ValueError('malformed node or string: ' + repr(node))
ValueError: malformed node or string: <_ast.Call object at 0x183118cba8>

## Version and main components

* Spyder Version: 3.2.4 
* Python Version: 3.6.3
* Qt Versions: 5.6.2, PyQt5 5.6 on Darwin

## Dependencies
```
pyflakes >=0.6.0 :  1.6.0 (OK)
pycodestyle >=2.3:  2.3.1 (OK)
pygments >=2.0   :  2.2.0 (OK)
pandas >=0.13.1  :  0.20.3 (OK)
numpy >=1.7      :  1.13.3 (OK)
sphinx >=0.6.6   :  1.6.3 (OK)
rope >=0.9.4     :  0.10.5 (OK)
jedi >=0.9.0     :  0.10.2 (OK)
psutil >=0.3     :  5.4.0 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
sympy >=0.7.3    :  1.1.1 (OK)
cython >=0.21    :  0.26.1 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
IPython >=4.0    :  6.1.0 (OK)
pylint >=0.25    :  1.7.4 (OK)

```

**Comment by ccordoba12** [comment]
This is fixed in our latest version (3.2.8). Please update by opening Terminal.app and running there

    conda update spyder

**Comment by CAM-Gerlach** [comment]
Duplicate of #5636 , fixed in Spyder 3.2.6
